scripts/bootstrap.sh

```diff
@@ -87,13 +87,13 @@ then
             osInfo[/etc/centos-release]="sudo yum -y install"
             osInfo[/etc/fedora-release]="sudo dnf -y install"
             package_manager=""
-            for f in ${!osInfo[@]}
```

Comment:
the error shellcheck gave me here was a bit weird. I'd like @ljupcovangelski to have a closer look